light-node/Cargo.toml

```diff
@@ -9,6 +9,10 @@ publish = false
 abscissa_tokio = "0.5"
 async-trait = "0.1"
 gumdrop = "0.7"
+jsonrpc-core = "14.2"
+jsonrpc-core-client = "14.2"
+jsonrpc-http-server = "14.2"
+jsonrpc-derive = "14.2"
```

Comment:
do we plan to derive rpc functionality from exiting structs / methods (I'm all for it but that means less control over the implementation I guess).

Reply:
The upside of that approach is that you get a client and a server. Not fuzzed either way, just thought it would be helpful for utilitiies that operate against the running API.